internal/host/hostcommands/install_cmd.go

```diff
@@ -150,10 +147,14 @@ func (i *installCmd) getFullInstallerCommand(cluster *common.Cluster, host *mode
 	if hostInstallerArgs != "" {
 		request.InstallerArgs = hostInstallerArgs
 	}
+
 	if hostutil.SaveDiskPartitionsIsSet(hostInstallerArgs) {
 		request.SkipInstallationDiskCleanup = true
+		host.SkipFormattingDisks = strings.Join(append(strings.Split(host.SkipFormattingDisks, ","), bootdevice), ",")
```

Comment:
Why string manipulation on `host.SkipFormattingDisks` instead of removal from `request.DisksToFormat`? Manipulating arrays is more intuitive

Also please move this logic to a helper `removeDiskFromArray()` helper function because go syntax for  array / string manipulation for removing from an array is not intuitive at all and very hard to read and it's hard to understand from a quick look that this is what you're doing here

Reply:
didn't wanted to do it actually. Didn't see any need for new function.
Another thing is in this case we will not have event on skipping this disk.
So to say the truth i sort of prefer using adding it to string in order to have skipping logic on it

Reply:
I can make this string manipulation easier 
if host.SkipFormattingDisks != "" {
   host.SkipFormattingDisks += fmt.Sprintf(",%s", bootdevice)
} else {
   host.SkipFormattingDisks = bootdevice
}

more readable maybe?

Reply:
Yes